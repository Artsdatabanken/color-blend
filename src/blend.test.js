const blandFarger = require("./blend");

describe("fargefunksjon", () => {
  it("mix saturation and color", () => {
    const farger = [
      { farge: "hsl(320,60%, 70%)", vekt: [0, 1, 0] },
      { farge: "hsl(120,90%, 30%)", vekt: [1, 0, 1] }
    ];
    const actual = blandFarger(farger);
    expect(actual).toMatchSnapshot();
  });
});
