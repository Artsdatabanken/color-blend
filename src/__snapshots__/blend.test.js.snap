// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fargefunksjon mix saturation and color 1`] = `"hsl(120, 60%, 30%)"`;
